### PR TITLE
Updates to submissions do not recreate D files

### PR DESCRIPTION
**Pull request by jworcestBAH**
Fixes the bug of hanging after updates on cross-file

### CODE REVIEW

File: dataactbroker/handlers/fileHandler.py [hunk 1]
```diff
@@ -164,18 +164,20 @@ def submit(self,name,CreateCredentials):
...
-                    filename = CONFIG_BROKER["d1_file_name"]
-
-                if(not self.isLocal):
-                    uploadName = str(name) + "/" + s3UrlHandler.getTimestampedFilename(filename)
-                else:
-                    uploadName = filename
-                responseDict[extFileType + "_key"] = uploadName
-                fileNameMap.append((extFileType, uploadName, filename))
+            if not existingSubmission:
+                # Existing submissions do not recreate D jobs
```
Comment (bsweger) [inline dataactbroker/handlers/fileHandler.py]: Thanks for the comment!

**Comment by bsweger** [comment]
@jworcestBAH Done reviewing. Looks good--just have one question about existing submissions. If a user needs to modify a submission and has to regenerate the D files as a result, is that something we currently handle, or is that future story? If the former, does this new logic accommodate that scnenario?

**Comment by jworcestBAH** [comment]
Regeneration of D files works by calling the generate_d2_file and check_d2_file routes, that process is unaffected by this change.  By not resetting the statuses here, the user still has the option of regenerating the D2 file on the cross-file page, but we do not require the d2 file to be regenerated whenever a submission is changed (which the frontend does not support anyway).

**Comment by bsweger** [comment]
Thanks for the context! Looks good..merging!

**Pull request merged**